lib/recurly/element/element.js

```diff
@@ -130,6 +130,7 @@ export default class Element extends Emitter {
     iframe.setAttribute('name', `${FRAME_NAME_PREFIX}${id}`);
     iframe.setAttribute('allowpaymentrequest', 'true');
     iframe.setAttribute('style', 'background: none; width: 100%; height: 100%;');
+    if (this.iframeTitle) iframe.setAttribute('title', this.iframeTitle);
```

Comment:
Are single line conditionals without curly braces idiomatic JS?

Reply:
It matches the style of the rest of the project and passes linting, so it's okay.